internal/controller/etcdcluster_controller_test.go

```diff
@@ -148,21 +179,21 @@ var _ = Describe("EtcdCluster Controller", func() {
 				Expect(err).ToNot(HaveOccurred())
 			})
 
-			By("setting owned StatefulSet to ready state", func() {
-				Eventually(Get(&statefulSet)).Should(Succeed())
-				Eventually(UpdateStatus(&statefulSet, func() {
-					statefulSet.Status.ReadyReplicas = *etcdcluster.Spec.Replicas
-					statefulSet.Status.Replicas = *etcdcluster.Spec.Replicas
-				})).Should(Succeed())
-			})
-
-			By("reconciling the EtcdCluster after owned StatefulSet is ready", func() {
-				_, err = reconciler.Reconcile(ctx, reconcile.Request{NamespacedName: client.ObjectKeyFromObject(&etcdcluster)})
-				Expect(err).ToNot(HaveOccurred())
-				Eventually(Get(&etcdcluster)).Should(Succeed())
-				Expect(etcdcluster.Status.Conditions[1].Type).To(Equal(etcdaenixiov1alpha1.EtcdConditionReady))
-				Expect(string(etcdcluster.Status.Conditions[1].Status)).To(Equal("True"))
-			})
+			// By("setting owned StatefulSet to ready state", func() {
```

Comment:
Sorry for this commenting. I wrote my opinion in the PR description. As a tradeoff to uncomment it: we can define environment variable `AUTOTESTS: true` that will be used in functions that can't be executed during autotests:
```
if os.Getenv("AUTOTESTS") == "true" {
  return
}
```

I really do not want to overcomplicate tests with multiple mocks.

Reply:
Agreed to uncomment and skip this test, because the same test is done on e2e level every time

Reply:
Done